scripts/windows/build/Create-ArmBase.ps1

```diff
@@ -0,0 +1,78 @@
+<#
+ # This script creates an ARM docker image as a base for EdgeAgent, EdgeHub, and general module.
+ # then pushes it to the appropriate registries.
+ # It assumes that the caller is logged into registries:
+ # edgebuilds.azurecr.io
+ # edgerelease.azurecr.io
+ # hub.docker.com
+ #>
+ 
+param (
+    [ValidateNotNullOrEmpty()]
+    [ValidateScript( {Test-Path $_ -PathType Container})]
+    [String] $ProjectDirectory,
+    
+    [ValidateNotNullOrEmpty()]
+    [String] $DockerNamespace,
+    
+    [ValidateSet("azureiotedge-module-base", "azureiotedge-hub-base", "azureiotedge-agent-base")]
+    [String] $DockerImageName,
+    
+    [ValidateNotNullOrEmpty()]
+    [String] $DockerImageVersion,
+
+    [Switch] $NoPush
+)
+
+$DockerImageArchitecture = "arm32v7"
+
+<#
+ # Docker file
+ #>
+$DockerFile = "$ProjectDirectory/docker/windows/$DockerImageArchitecture/base/Dockerfile"
+if (-Not (Test-Path $DockerFile -PathType Leaf))
+{
+    throw "$DockerFile not found"
+}
+
+<#
+ # Docker registries
+ #>
+$DockerRegistries = New-Object 'System.Collections.Generic.List[String]'
```

Comment:
Wouldn't `@("edgebuilds.azurecr.io/", "edgerelease.azurecr.io/")` be good enough?